pwndbg/arguments.py

```diff
@@ -104,6 +105,23 @@ def get(instruction: PwndbgInstruction) -> List[Tuple[pwndbg.lib.functions.Argum
 
     if func:
         args = func.args
+        if args[-1].name == "vararg":
+            args.pop()
+            if len(args) > 0:
+                format_value = pwndbg.enhance.enhance(argument(len(args) - 1, abi))
+                vararg_cnt = 0
+                try:
+                    vararg_cnt = int(format_value)
+                except ValueError:
+                    m = re.findall(
+                        r"%[-+ #0]?(?:[0-9]+|\*)?(?:\.(?:[0-9]+|\*))?(?:hh|h|ll|l|j|z|t|L)?[diuoxXfFeEgGaAcspn]",
```

Comment:
missing a few non-standard specifiers that are mentioned in https://www.man7.org/linux/man-pages/man3/printf.3.html 
```suggestion
                        r"%[-+ #0]?(?:[0-9]+|\*)?(?:\.(?:[0-9]+|\*))?(?:hh|h|l|ll|q|L|j|z|Z|t)?[diuoxXfFeEgGaAcsCSpn]",
```